server/routes.go

```diff
@@ -1431,9 +1431,11 @@ func (s *Server) ChatHandler(c *gin.Context) {
 
 	r, m, opts, err := s.scheduleRunner(c.Request.Context(), req.Model, caps, req.Options, req.KeepAlive)
 	if errors.Is(err, errCapabilityCompletion) {
+		slog.Error("capability error", "error", err)
 		c.JSON(http.StatusBadRequest, gin.H{"error": fmt.Sprintf("%q does not support chat", req.Model)})
 		return
 	} else if err != nil {
+		slog.Error("schedule error", "error", err)
```

Comment:
added some err messaging here instead of just being in the JSON return

Reply:
I'd probably keep these out of the scope of this PR (perhaps in a better logging PR)

Reply:
Also, we should reserve `slog.Error` for internal errors for monitoring – this is a valid error for users so we should probably not log at all in this case